contracts/imsv-funding/Master.algo.ts

```diff
@@ -581,13 +580,12 @@ export class Master extends Contract.extend(Ownable, Pausable) {
     /**
      * Retrieves the minimum balance requirement for creating a card fund account.
      *
-     * @param ref - Client reference to store on the Card Fund
      * @returns Minimum balance requirement for creating a card fund account
      */
-    getCardFundMbr(ref: string): uint64 {
+    getCardFundMbr(): uint64 {
         // Card Fund Data Box Cost:
         // 2500 + 400 * ((Prefix + Address) + ((partnerChannel + owner + address + nonce + withdrawalNonce + (ABIHead + (ABI encoded reference)))))
-        const cardFundDataBoxCost = 2500 + 400 * ((2 + 32) + ((32 + 32 + 32 + 8 + 8 + (2 + (2 + ref.length)))));
+        const cardFundDataBoxCost = 2500 + 400 * ((2 + 32) + ((32 + 32 + 32 + 8 + 8 + (2 + (2 + 64)))));
```

Comment:
The comment and calculation need to be updated after changing a dynamic length string to a fixed length byte array.
You can remove the "ABIHead" part which provides the offset to the dynamic array, reducing the value by 2 (or 800 uAlgo).

Reply:
Whilst I think about it, if this is to be a static value MBR requirement (like is required in `cardFundDeployInit`), could it not just be documented as such and avoid the read-only function all together, since it's never going to be different?